### PR TITLE
fixes-for-clip-conversion

### PR DESCRIPTION
**Pull request by RanHomri**
- modify convert_flatten to match onnx documentation
- enable unsqueeze of multiple axes

### CODE REVIEW

File: onnx2kerastl/reshape_layers.py [hunk 1]
```diff
@@ -323,11 +326,11 @@ def convert_flatten(node, params, layers, lambda_func, node_name, keras_name):
     logger.debug('Convert inputs to Keras/TF layers if needed.')
     input_0 = ensure_tf_type(layers[node.input[0]], name="%s_const" % keras_name)
 
-    # Fix critical issue with flatten
-    permute = keras.layers.Permute((3, 1, 2))
-    tensor_chw = permute(input_0)
-    flatten = keras.layers.Flatten(name=keras_name)
-    layers[node_name] = flatten(tensor_chw)
+    input_dims = input_0.shape
+    flatten_axis = params['axis']
```
Comment (doronHarnoy) [inline onnx2kerastl/reshape_layers.py:330]: I think sometimes axis will not be shown so we can use the default value
```suggestion
    flatten_axis = params.get('axis', 1)
```

File: onnx2kerastl/reshape_layers.py [hunk 2]
```diff
@@ -291,17 +291,20 @@ def convert_unsqueeze(node, params, layers, lambda_func, node_name, keras_name):
         else:
             raise AttributeError('Number of inputs is not equal 1 for unsqueeze layer')
 
+    if len(np.unique(params['axes'])) < len(params['axes']):
+        raise AttributeError(f"The specified axes contains duplicates values: {params['axes']}")
```
Comment (doronHarnoy) [inline onnx2kerastl/reshape_layers.py:295]: Did we see duplication of axes in a recent onnx model?

**Review by doronHarnoy — APPROVED** [review]
Cool

File: onnx2kerastl/reshape_layers.py (hunk 2 above)
Comment (RanHomri) [reply onnx2kerastl/reshape_layers.py:295]: no, but in onnx docs it's mention that this is not allowed so I added a check
